listenbrainz/webserver/templates/profile/info.html

```diff
@@ -28,4 +32,15 @@ <h3>User token</h3>
       <span class="btn btn-info btn-lg" style="width: 200px"><a href="{{ url_for("profile.reset_token") }}" style="color: white;">Reset token</a></span>
     </p>
 
+  </div>
+{% endblock %}
+
+{% block scripts %}
```

Comment:
Why do we need to add this block here?

Reply:
I misunderstood the template earlier. It isn't needed. Fixing it.